bodhi/server/models.py

```diff
@@ -1709,41 +1804,39 @@ def check_requirements(self, session, settings):
         return True, "All checks pass."
 
     def check_karma_thresholds(self, db, agent):
-        """
-        Check if we have reached either karma threshold, call set_request if necessary,
-        and Ignore karma thresholds if the update is locked and raise exception
-        """
-        # Raise Exception if the update is locked
-        if self.locked:
+        """Check if we have reached either karma threshold, and call set_request if necessary"""
```

Comment:
@ankit01ojha Looks like you do not have the recent updated code base. 
Please make sure to rebase upstream code base branch with your local branch.